indra/newview/llfloaterimsessiontab.cpp

```diff
@@ -76,7 +77,7 @@ LLFloaterIMSessionTab::LLFloaterIMSessionTab(const LLSD& session_id)
 	mInputPanels(NULL),
 	mChatLayoutPanelHeight(0)
 {
-    setAutoFocus(FALSE);
+	setAutoFocus(FALSE);
```

Comment:
Please make a separate out of formating fixes, it makes commit hard to review. Also we really shouldn't be replacing tabs with spaces, it's against coding policy.